code/game/objects/effects/temporary_visuals/miscellaneous.dm

```diff
@@ -364,6 +364,10 @@
 	icon_state = "impact_laser_purple"
 	duration = 4
 
+/obj/effect/temp_visual/impact_effect/yellow_laser
+	icon = 'icons/obj/projectiles.dmi'
```

Comment:
? this is either inherited from parent or you put that laser icon in the wrong file

Reply:
Yeah ill fix it I was just lazy